en/docs/references/postman-integration.md

```diff
@@ -0,0 +1,129 @@
+# Test Your Service via Postman
+
+The Postman Editor allows you to integrate with your Postman account to easily test your services.
+
+## Prerequisites
+
+In order to use the Postman Editor:
+
+- Make sure you have logged into the Choreo console and created a service. For instructions to create a service, see [Create Your First Service](../services/create-your-first-service.md).
+
+    !!! tip
+        This guide uses the service created in [Create Your First Service](../services/create-your-first-service.md) as an example.
+        
+- Have an active Postman account.
+
+## Step 1: Generate Postman API key
+
+To generate the Postman API key so that you can connect your Postman account to Choreo, follow this procedure:
+
+1. Sign in to your Postman account.
+
+2. Go to your profile, and click **Account Settings**. Then click **Postman API Keys**
+
+    ![Generate Postman API key](../assets/img/postman/generate-postman-api-key.png){.cInlineImage-full}
```

Comment:
Isn't `{.cInlineImage-half}` more suitable for this?

Reply:
Also is this image really useful to the user? If we really need to have it get the screenshot after selecting `Postman API Keys` instead of above.

<img width="986" alt="image" src="https://user-images.githubusercontent.com/16300038/124820125-b203b380-df8a-11eb-94f9-774d75aeed13.png">

Reply:
Removed the image.